src/main/resources/org/sonatype/nexus/ci/nxrm/v3/CreateTagBuilder/help-tagAttributesPath.html

```diff
@@ -0,0 +1,15 @@
+<!--
+
+    Copyright (c) 2016-present Sonatype, Inc. All rights reserved.
+
+    This program is licensed to you under the Apache License Version 2.0,
+    and you may not use this file except in compliance with the Apache License Version 2.0.
+    You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the Apache License Version 2.0 is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+
+-->
+The file path (environment-aware) to the optional metadata for the tag, using the workspace as the base path
```

Comment:
Is there a reason the other helps use a `<div/>` and this one doesn't?

Reply:
I was curious about this as well

Reply:
No real reason other than to assist in text wrap. Simple enough to add uniformity (and wrap in a div) so I'll do that